workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/InviteMemberModal/InviteMemberModal.tsx

```diff
@@ -161,14 +162,22 @@ export function InviteMemberModal({
     await addExistentUserIntoAccount(selectedAccount.id, formFields.email);
     await addAccountRolesToUser(user);
 
+    const accountGroups = await getAccountGroup(selectedAccount.id);
+
+    let accountGroupERC: string = "";
+    if (accountGroups?.length > 0) {
+      accountGroupERC = accountGroups[0]?.externalReferenceCode;
```

Comment:
Why this` accountGroups[0]`?

Could we use 
`const accountGroupERC = accountGroups && accountGroups[0]?.externalReferenceCode`?